dataland-csvconverter/src/main/kotlin/org/dataland/csvconverter/csv/commonfieldparsers/FiscalYearParser.kt

```diff
@@ -27,16 +27,16 @@ class FiscalYearParser {
     /**
      * Method to retrieve information about the deviation of the fiscal year from the csv file
      */
-    fun getFiscalYearDeviation(csvLineData: Map<String, String>): FiscalYearDeviation {
-        return columnMappingFiscalYear.getCsvValue("fiscalYearDeviation", csvLineData)
-            .let { fiscalYearDeviationParser.parse("fiscalYearDeviation", it) }
+    fun getFiscalYearDeviation(csvLineData: Map<String, String>): FiscalYearDeviation? {
```

Comment:
Is this due to not availaible information about fiscal year deviation / fiscal year end?